js/ethereum-utils/src/signature.ts

```diff
@@ -128,6 +138,10 @@ function getTypesFor(payloadType: string): SupportedPayloadDefinitions {
     AddKeyData: ADD_KEY_DATA_DEFINITION,
     AuthorizedKeyData: AUTHORIZED_KEY_DATA_DEFINITION,
     AddProvider: ADD_PROVIDER_DEFINITION,
+
+    // offchain signatures
```

Comment:
We'll need to add the one for stateful storage and handles, although A. this is just the initial draft, and B. those are close to the `SIWF_SIGNED_REQUEST_PAYLOAD_DEFINITION`